x-pack/plugins/enterprise_search/public/applications/analytics/components/analytics_collection_view/analytics_collection_explore_table_formulas.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { TimeRange } from '@kbn/data-plugin/common';
+
+export const getSearchQueryRequestParams = (search: string): { include?: string } => {
+  const createRegexQuery = (queryString: string) => {
+    let query = queryString.replace(/[\-\[\]\/\{\}\(\)\*\+\?\.\\\^\$\|]/g, '\\$&');
+    query = query
+      .split('')
+      .map((char) => {
+        if (/[a-z]/.test(char)) {
```

Comment:
question: Do we need to deal with special and/or non-Roman alphabet characters? Cyrillic, chinese, or any character with diacritics?